.github/workflows/prerelease.yml

```diff
@@ -8,65 +8,9 @@ on:
       - 'v*'
 
 env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  NRJMX_VERSION: '2.6.0'
-  INTEGRATION: "jmx"
-  ORIGINAL_REPO_NAME: 'newrelic/nri-jmx'
-  REPO_FULL_NAME: ${{ github.event.repository.full_name }}
-  TAG: ${{ github.event.release.tag_name }}
+  NRJMX_VERSION: '2.6.0' # TODO check if we should update it to latest release version
```

Comment:
I think it is important to understand how this is used. I don't see any reference in the `prerelease.yml` and `push_pr.yml` files, but I expect that it is used in one on the underlying scripts.

Can we check its usage and that its value is not lost when using reusable workflows?

Reply:
https://github.com/search?q=repo%3Anewrelic%2Fnri-jmx%20NRJMX_VERSION&type=code